code/datums/gamemode/role/wizard_apprentice.dm

```diff
@@ -0,0 +1,69 @@
+/datum/role/wizard_master
+	name = "master wizard"
+	id = WIZAPP_MASTER
+	disallow_job = TRUE
+	logo_state = "wizard-logo"
+	greets = list(GREET_DEFAULT)
```

Comment:
same greet thing

Reply:
Removed it entirely since you are never supposed to be greeted to this role anyway